testing/engine/multipart.go

```diff
@@ -42,7 +42,7 @@ airween
 `,
 						},
 						Output: profile.ExpectedOutput{
-							TriggeredRules:    []int{100},
+							TriggeredRules:    []int{100, 999997, 999998, 999999},
```

Comment:
I would not use rule IDs above 50000. In particular, CRS uses ID 999999 for tests.